uefi/src/table/runtime.rs

```diff
@@ -290,6 +292,23 @@ impl RuntimeServices {
     ) -> Status {
         (self.0.set_virtual_address_map)(map_size, desc_size, desc_version, virtual_map)
     }
+
+    /// Passes capsules to the firmware. Capsules are most commonly used to update the firmware
+    /// or for by the operating system to have information persist across a system reset.
```

Comment:
typo: `or for by the`. I think you could just drop the 2nd half though, since I'm not sure I believe the spec when it says OSes commonly use capsules to persist information. So, suggest shortening to:

```
    /// Passes capsules to the firmware. Capsules are most commonly used to update system firmware.
```